src/Pim/Bundle/VersioningBundle/spec/Normalizer/Flat/ProductNormalizerSpec.php

```diff
@@ -73,10 +73,10 @@ function it_normalizes_product(
 
         $this->normalize($product, 'flat', [])->shouldReturn(
             [
-                'sku'        => 'sku-001',
```

Comment:
I did this too (already merged on TIP-613), but in fact it is wrong.
We broke a PIM features: having the sku first. You can see it on exports, the sku is in the middle instead being on the first column.
(but the fact is, I have no idea why it is this way…)